Documentation/spelling_wordlist.txt

```diff
@@ -4,6 +4,7 @@ Alemayhu
 Alibaba
 Aquantia
 Attestor
+AWS
```

Comment:
"aws" (lowercase) is already in the list. Lowercase entries should be case-insensitive, so uppercase "AWS" shouldn't be needed. Did the spell check fail without it?

Reply:
No, it passed.
I have removed the uppercase "AWS".